tileserver/__init__.py

```diff
@@ -261,6 +261,7 @@ def handle_request(self, request):
             formatted_tiles, extra_data = format_coord(
                 coord,
                 nominal_zoom,
+                nominal_zoom+1,
```

Comment:
Probably this should be 16 instead, please give that a try and let me know if you still get your expected results.

See also:

- https://github.com/tilezen/tilequeue/pull/402/files#diff-c63c99a87f47c094869dcec4c787506ddf92c8394f99b24fc32ccd5746a05bd9R784

Reply:
(That's for a typical zoom 15 (512px size) max_zoom... so if you want zoom 14 you may need to try also setting this locally to 15 instead... but the PR here should be for the generic case.)

Reply:
Yes, I have previously tried to hardcode there `16` instead of `nominal_zoom+1` but I did not get expected results. For me it seems that there should be `17`.

Reply:
it's been a while since I've been familiar with the details, but I expect that nominal zoom (display zoom) is actually what we want here. If features are not appearing, could it be because the features on those properties aren't set appropriately? (Treat this with a grain of salt since I don't have the context)